pdns/syncres.cc

```diff
@@ -1099,29 +1100,31 @@ bool SyncRes::doCacheCheck(const DNSName &qname, const QType &qtype, vector<DNSR
 
       vState recordState = getValidationStatus(qname, false);
       if (recordState == Secure) {
-        LOG(prefix<<sqname<<": got Indeterminate state from the cache, validating.."<<endl);
+        DNSSECLOG(prefix<<sqname<<": got Indeterminate state from the cache, validating.."<<endl);
         cachedState = SyncRes::validateRecordsWithSigs(depth, sqname, sqt, sqname, cset, signatures);
 
         if (cachedState != Indeterminate) {
-          LOG(prefix<<qname<<": got Indeterminate state from the cache, validation result is "<<vStates[cachedState]<<endl);
+          DNSSECLOG(prefix<<qname<<": got Indeterminate state from the cache, validation result is "<<vStates[cachedState]<<endl);
           t_RC->updateValidationStatus(d_now.tv_sec, sqname, sqt, d_incomingECSFound ? d_incomingECSNetwork : d_requestor, d_requireAuthData, cachedState);
         }
       }
     }
 
+    DNSSECLOG(prefix<<qname<<" DNSSEC validated the following record content:"<<endl);
```

Comment:
This message seems misleading, the content might not have been validated.

Reply:
Can you explain this some more. I might have misinterpreted the code here.

Reply:
If I'm not mistaken we will log the content of every record retrieved from the cache, be it validated or not. It might not have been validated if validation was not requested, or the cached data was not authoritative, for example.

Reply:
ok, then this should not be DNSSECLOG but LOG (with a better message)